src/App.tsx

```diff
@@ -1,26 +1,272 @@
+      title: newTodoTitle,
+      completed: false,
+      userId: USER_ID,
+    };
+
+    setTempTodo(tempTodoItem); // Додаємо тимчасову тудушку у стейт
+    setIsSubmitting(true);
+
+    // Відправляємо запит на створення тудушки
+    createTodo({
+      title: newTodoTitle,
+      userId: USER_ID,
+      completed: false,
+    })
+      .then(newTodo => {
+        setTodos(currentTodos => [...currentTodos, newTodo]); // Додаємо нову тудушку у список
+        setTempTodo(null); // Видаляємо тимчасову тудушку після отримання відповіді
+        setNewTodoTitle(''); // Очищаємо поле вводу
+      })
+      .catch(() => setErrorMessage('Unable to add todo'))
+      .finally(() => setIsSubmitting(false));
+  };
+
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">
-          React Todo App - Load Todos
-        </a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          <button
+            type="button"
+            className={classNames('todoapp__toggle-all', {
+              active: hasAllTodosCompleted,
+            })}
+            data-cy="ToggleAllButton"
+          />
+
+          {/* Форма для добавления новой задачи */}
+          <form onSubmit={handleAddTodo}>
+            <input
+              data-cy="NewTodoField"
+              type="text"
+              className="todoapp__new-todo"
+              placeholder="What needs to be done?"
+              value={newTodoTitle}
+              onChange={event => setNewTodoTitle(event.target.value)}
+              ref={inputRef}
+              disabled={isSubmitting}
+            />
+          </form>
+        </header>
+
+        <section className="todoapp__main" data-cy="TodoList">
+          {filteredTodos.map(todo => (
+            <div
+              key={todo.id}
+              data-cy="Todo"
+              className={classNames('todo', { completed: todo.completed })}
+            >
+              <label className="todo__status-label">
+                <input
+                  data-cy="TodoStatus"
+                  type="checkbox"
+                  className="todo__status"
+                  checked={todo.completed}
+                />
+              </label>
+
+              <span data-cy="TodoTitle" className="todo__title">
+                {todo.title}
+              </span>
+
+              <button
+                type="button"
+                className="todo__remove"
+                data-cy="TodoDelete"
+                onClick={() => handleDeleteTodo(todo.id)}
+              >
+                ×
+              </button>
+            </div>
```

Comment:
You have to use Loader on Todo as well to pass test, but you don't have to show it (just remove class is-active)